src/biogeochem/CNDriverMod.F90

```diff
@@ -347,7 +364,8 @@ subroutine CNDriverNoLeaching(bounds,
     ! When nutrient competition is required to be done at cohort level both plant_nutrient_demand and 
     ! do_nutrient_competition should be modified, but that modification should not significantly change 
     ! the current interface.
-
+!     if(begp .le. 8428 .and. endp .ge. 8428)then
+!     end if
```

Comment:
Please remove the above commented out lines.

Reply:
done